crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -220,7 +219,6 @@ pub struct Solution {
     prices: HashMap<eth::H160, eth::U256>,
     trades: Vec<Trade>,
     interactions: Vec<Interaction>,
-    score: Option<eth::U256>,
```

Comment:
In CIP-20, the solver engines provide a "risk" value to the solution they propose (called "score discount" in the CIP AFAIR). This flat value is subtracted from the driver-computed score and represents how much risk the solver thinks the transaction has.

I'm fine with including this in a separate PR, but it is needed for full CIP-20 support.

Reply:
https://github.com/cowprotocol/services/pull/1526/commits/c94cfe099feecf2f1eaf3559a5f822823ada790f